crates/release-automation/src/lib/crate_selection/forest.rs

```diff
@@ -54,8 +54,14 @@ pub fn flatten_forest<'a>(crates: &'a Vec<Crate<'a>>) -> Fallible<Vec<&'a Crate<
         );
     }
 
-    if crates.len() != order.len() {
-        bail!("While attempting to order crates, managed to order {} of {} crates. This likely means there is a cyclic dependency.", order.len(), crates.len());
+    let num_crates = crates.len();
```

Comment:
This looks useful for debugging but I'd really prefer to review changes to the `release-automation` separately

Reply:
I can break this out

Reply:
https://github.com/holochain/holochain/pull/3599